packages/flutter/lib/src/widgets/view.dart

```diff
@@ -96,6 +106,174 @@ class View extends StatelessWidget {
+  /// responsible for managing the render tree of that view. If there is no
+  /// [View] ancestor, [RendererBinding.rootPipelineOwner] is returned instead.
+  static PipelineOwner pipelineOwnerOf(BuildContext context) {
+    return context.dependOnInheritedWidgetOfExactType<_PipelineOwnerScope>()?.pipelineOwner
+        ?? RendererBinding.instance.rootPipelineOwner;
+  }
+
+  @override
+  State<View> createState() => _ViewState();
+}
+
+class _ViewState extends State<View> with WidgetsBindingObserver {
+  final FocusScopeNode _scopeNode = FocusScopeNode(
+    debugLabel: kReleaseMode ? null : 'View Scope',
+  );
+  final FocusTraversalPolicy _policy = ReadingOrderTraversalPolicy();
+
+  @override
+  void initState() {
+    super.initState();
+    WidgetsBinding.instance.addObserver(this);
+  }
+
+  @override
+  void dispose() {
+    WidgetsBinding.instance.removeObserver(this);
+    _scopeNode.dispose();
+    super.dispose();
+  }
+
+  @override
+  void didChangeViewFocus(ViewFocusEvent event) {
+    if (event.viewId != widget.view.viewId) {
+      // The event is not pertinent to this view.
+      return;
+    }
+    FocusNode nextFocus;
+    switch (event.state) {
+      case ViewFocusState.focused:
+        switch (event.direction) {
+          case ViewFocusDirection.forward:
+            nextFocus = _policy.findFirstFocus(_scopeNode, ignoreCurrentFocus: true) ?? _scopeNode;
+          case ViewFocusDirection.backward:
+            nextFocus = _policy.findLastFocus(_scopeNode, ignoreCurrentFocus: true);
+          case ViewFocusDirection.undefined:
+            nextFocus = _scopeNode;
+        }
+        nextFocus.requestFocus();
+      case ViewFocusState.unfocused:
+        // Focusing on the root scope node will "park" the focus, so that no
+        // descendant node will be given focus, and there's no widget that can
+        // receive keyboard events.
+        FocusManager.instance.rootScope.requestScopeFocus();
+    }
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return RawView(
+      view: widget.view,
+      deprecatedDoNotUseWillBeRemovedWithoutNoticePipelineOwner: widget._deprecatedPipelineOwner,
+      deprecatedDoNotUseWillBeRemovedWithoutNoticeRenderView: widget._deprecatedRenderView,
+      child: MediaQuery.fromView(
+        view: widget.view,
+        child: FocusTraversalGroup(
+          policy: _policy,
+          child: FocusScope.withExternalFocusNode(
+            focusScopeNode: _scopeNode,
+            child: widget.child,
+          ),
+        ),
+      ),
+    );
+  }
+}
+
+/// The lower level workhorse widget for [View] that bootstraps a render tree
+/// for a view.
+///
+/// The [RawView] widget is typically used when it is not desirable to have a
+/// [FocusScope] or [MediaQuery] around the resulting widget tree, in the way a
+/// [View] creates.
+///
+/// This widget is typically used at the root of the widget tree outside of any
```

Comment:
maybe remove the "typically" here and just say that it can be used here. (again to emphasize that using this is actually not that typical)

Reply:
Done.